### PR TITLE
Proper Syntax error over global redefinition

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

File: pythran/transformations/expand_globals.py [hunk 1]
```diff
@@ -46,6 +48,10 @@ def visit_Module(self, node):
                 continue
             for target in stmt.targets:
                 assert isinstance(target, ast.Name)
+                if target.id in self.to_expand:
+                    raise PythranSyntaxError(
```
Comment (pbrunet) [inline pythran/transformations/expand_globals.py]: You raise an error? Not just a warning?

Comment (pbrunet) [inline pythran/transformations/expand_globals.py]: BTW, should we also check it with function with the same name?

Comment (serge-sans-paille) [inline pythran/transformations/expand_globals.py]: yes, taht's what we already do for function redefinition anyway.

Comment (serge-sans-paille) [inline pythran/transformations/expand_globals.py]: That's already done. in fact the code already rasied an error, but
blaming a function redefinition as globals are turned into functions.

**Pull request merged**